src/Orchestrator/Controllers/StatusController.cs

```diff
@@ -151,6 +151,7 @@ public async Task<ActionResult<NetAppStatusModel>> GetNetAppStatusesAsync(bool g
         try
         {
             var status = await _heartbeatService.GetAllAppStatusesAsync(generateFakeData);
+            if (status == null) return BadRequest();

```

Comment:
@appmdev why duplicate the null check on statuses?
In the next line, we are already checking for null and returning `NotFound`. It is unnecessary to have it here.